### DISCUSSION TITLE
Architectural Planning and Project Management Components

### DISCUSSION THREAD

**Discussion by jpl-jengelke** [NASA-AMMOS/slim#22] [Ideas]
Has there been any consideration of the architecture or project management part of SLIM? For instance, what diagrams or planning guides are required before starting a project? UML? Is there any PM guidance, such as Agile or a light-version of Agile?

**Comment by riverma** [comment]
Hey @jpl-jengelke - I don't think that's really been mentioned much within SLIM to my knowledge. There are often some formal requirements for a software design description (SDD) or an architectural specification for many mission projects internally at NASA-JPL.

There's also the NASA SE Handbook (linked to within the [Documentation Use Cases diagram](https://nasa-ammos.github.io/slim/documentation/use-cases/markmap.html)) that has recommendations on design work: https://www.nasa.gov/seh/4-design-process
